docs/sops/serverless-controller-knative-up.md

```diff
@@ -0,0 +1,149 @@
+# OpenShift Serverless Knative Serving Control Plane Readiness
+
+## Severity: Critical
+
+## Impact
+
+Any users or services using Knative Serving will be unable to get their Knative Services reconciled.
+This means, any new Knative Services might not be created, and any existing Knative Services might not be updated. Also, it might also mean that the user workload is not being served.
+
+## Summary
+
+OpenShift Serverless does not create any alerts out-of-the box. However, users can create alerts based on the gauges and metrics exposed.
+
+The gauge `knative_up{namespace="openshift-serverless", type="serving_status"}` should always have a value of `1`.  You should create an alert as described in https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0/ for using this gauge and measure it over some period of time.
+
+This failure should only occur in extremely catastrophic scenarios as all components are deployed in HA configuration.
+
+Possible causes can be:
+- Network issues in the cluster
+- Resource starvation on the cluster
+- OpenShift Serverless upgrade issues
+
+You should check the control plane status of OpenShift Serverless operator, Knative Serving and the used Knative Serving Ingress.
+
+## Prerequisites
+
+1. You must have admin access to the cluster via `oc` CLI.
+
+2. You must find out which Knative Serving Ingress is used.
+
+Run the following command to see the Knative Serving spec's ingress related section:
+
+```shell
+$ k get knativeserving -n knative-serving knative-serving -o jsonpath='{.spec.ingress}'
+```
+
+If the output is empty, then Kourier is used as the Knative Serving Ingress. 
+
+If the output is not empty, check which Ingress is used by looking at which ingress has `enabled: true`:
+
+```yaml
+  contour:
+    enabled: false
+  istio:
+    enabled: true
+  kourier:
+    enabled: false
+```
+
+In this case, Istio is used as the Knative Serving Ingress.
```

Comment:
Probably we should mention that we use ServiceMesh and people should also check their setup there for issues.

Reply:
I will make some adjustments to the text here.

Reply:
I mentioned ServiceMesh too now.

>  people should also check their setup there for issues
This is already written in step 11 below